### PR TITLE
Cache definitions in redis

### PR DESCRIPTION
**Pull request by simonvadee**

### CODE REVIEW

File: server/src/fhir/definitions.ts [hunk 1]
```diff
@@ -1,38 +1,64 @@
...
 
 // Gets a definition from the cache.
-export const getDefinition = (key: string): StructureDefinition | undefined =>
-  cache.get(key)
+export const getDefinition = async (
+  key: string,
+): Promise<StructureDefinition | undefined> => {
+  const { get } = cache()
+  const res = await get(key)
+  return res ? JSON.parse(res) : undefined
```
Comment (Jasopaum) [inline server/src/fhir/definitions.ts]: Are you doing this because we can't put objects in the cache?

File: server/src/resolvers/Query.ts [hunk 2]
```diff
@@ -101,7 +102,18 @@ export const Query = queryType({
         definitionId: idArg({ nullable: false }),
       },
       nullable: true,
-      resolve: (_, { definitionId }) => ({ id: definitionId }),
+      resolve: async (_, { definitionId }) => {
+        console.error('before')
```
Comment (Jasopaum) [inline server/src/resolvers/Query.ts]: remove this

File: server/src/resolvers/Query.ts [hunk 3]
```diff
@@ -101,7 +102,18 @@ export const Query = queryType({
...
+        const def = await getDefinition(definitionId)
+        console.error(!!def)
```
Comment (Jasopaum) [inline server/src/resolvers/Query.ts]: and this

File: server/src/fhir/definitions.ts (hunk 1 above)
Comment (simonvadee) [reply server/src/fhir/definitions.ts]: yes!

File: server/src/resolvers/Query.ts (hunk 2 above)
Comment (simonvadee) [reply server/src/resolvers/Query.ts]: done

File: server/src/resolvers/Query.ts (hunk 3 above)
Comment (simonvadee) [reply server/src/resolvers/Query.ts]: done

**Pull request merged**